proto/boxes_msg.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package benchmark_proto;
+message  Pose{
+    // position in world frame
+    Vector3 position = 1;
+
+    // orientation of boyd wrt world frame
+    Vector3 orientation = 2;
```

Comment:
I think a quaternion would be better here, similar to [gz::msgs::Pose](https://github.com/gazebosim/gz-msgs/blob/gz-msgs10/proto/gz/msgs/pose.proto#L39) (see also https://github.com/gazebosim/gz-msgs/blob/gz-msgs10/proto/gz/msgs/quaternion.proto)

and you should be able to use gz-math7 python bindings in the postprocessing script for the quaternion math